### DISCUSSION TITLE
Why there is so little information about the ghcr package?

### DISCUSSION THREAD

**Discussion by ackvf** [mckaywrigley/chatbot-ui#1379] [Help]
Looking in the "packages" https://github.com/mckaywrigley/chatbot-ui/pkgs/container/chatbot-ui
The only instruction there is:
```
docker pull ghcr.io/mckaywrigley/chatbot-ui:main
```
But this is the only time this package is ever mentioned. I was not sure what to do with it, trying to look up more information here and when I could not find anything, I just run it with default settings and to my surprise within few second the app was running on my Synology NAS.

Prior to that, I spent almost two hours figuring out how to make the steps from the readme https://github.com/mckaywrigley/chatbot-ui?tab=readme-ov-file#local-quickstart work on my Synology NAS to no avail, so my question is, why is this not mentioned in the readme?

**Comment by wizseek** [comment]
got this working in my synology too, at first had the same impression as you. But then I noticed the lack of features on the app and decided to check the [legacy version](https://github.com/mckaywrigley/chatbot-ui/tree/legacy).

It turns out that that docker image is very likely the package of legacy version.

**Reply by Stan-Stani** [reply]
I can definitely confirm that as of today it is the legacy version.

**Reply by ackvf** [reply]
Yeah, I am pretty sure it is the legacy version since it uses the localStorage instead of the db, but in all honesty, I prefer this setup, because it is much easier to modify the chat history this way and you basically have absolute control over everything.

Also the setup was just instant.

**Reply by wizlee** [reply]
100% agree that it is much easier. I ended up spending 2 days to get the current installation very close to working in my Synology. 
It is a lot of work as deploying using supabase CLI will have result in a working DB, I have to manually mirror what the CLI does into the open source supabase docker compose file. 

Now the only issue I have is regarding network (communication between containers) presumably because I missed configured one or more of the many supabase docker containers.

**Reply by Stan-Stani** [reply]
Unfortunately, I think current legacy system prompt setting via .env is broken and that's the main thing I want. Anyone interested in forking legacy and fixing it up with me?

**Reply by Stan-Stani** [reply]
I found out about the ghcr package from here: https://automation.baldacchino.net/building-a-private-chatgpt-interface-with-azure-openai/ and it works with Azure currently with those extra .env settings. It's odd that I can search this entire repo and cannot find references to those Azure specific .env settings except in some closed PRs. It looks like the maintainer scrubbed a fair amount of history from git. Or possibly the ghcr stuff was generated from open PRs that never actually got merged into main?

I've forked the legacy branch at its current state and it does not support the Azure .envs out of the box, so it seems legacy is a bit different from the legacy in the ghcr.
